modules/packages/Curl.chpl

```diff
@@ -168,10 +168,11 @@ module Curl {
   /* Returns the ``CURL`` handle connected to a channel opened with
```

Comment:
It looks like there's a doc link to `IO.channel` at the top of this file (line 49) - I don't know that we'll come back to it and I don't think we get warned about broken links, so it might be good to update it now